client/core/core.go

```diff
@@ -1655,6 +1655,7 @@ const defaultDEXPort = "7232"
 
 // addrHost returns the host or url:port pair for an address.
 func addrHost(addr string) (string, error) {
+	addr = strings.TrimSpace(addr)
```

Comment:
bizzare: https://play.golang.com/p/VL3fC6d8E_y

Reply:
> bizzare: https://play.golang.com/p/VL3fC6d8E_y

Interesting. Does did not qualify as a bug upstream?

Reply:
I don't know.  They'd probably say it has to be that way because the port part can be "service" like "ftp" or "dns" or whatever, so they don't validate whatever is after the last ":".